src/Components/home/componensts/Carousel.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+
+export const Carousel = ({activeIndex,prev,next,images}) => {
+    
+    return (
+        <div className="w-full">
+          <div className="relative w-full h-[530px] overflow-hidden rounded-lg">
+      
+            {images.map((url, index) => (
+              <img
+                key={index}
+                src={url}
+                alt={`Carousel Item ${index + 1}`}
+                className={`absolute w-full h-full transition-opacity duration-700 object-cover ease-in-out ${index === activeIndex ? "opacity-100" : "opacity-0"}`}
```

Comment:
please using this module instead of conditional checking https://www.npmjs.com/package/clsx